webviz_config/generic_plugins/_table_plotter.py

```diff
@@ -130,6 +131,18 @@ def plot_args(self) -> dict:
                     "multi": False,
                     "clearable": False,
                 },
+                "values": {
+                    "options": self.numeric_columns,
+                    "value": self.plot_options.get("values", None),
```

Comment:
Consider to set a default value for `values` to the first available numeric column, so that the user will actually see a pie chart when the plot type is selected? (Similar to `x`, `y` and `z`)
```suggestion
                    "value": self.plot_options.get("values", self.numeric_columns[0]),
```